src/Directory.Build.targets

```diff
@@ -44,7 +44,8 @@
 	<PackageReference Update="System.Numerics.Vectors" Version="4.5.0" />
 	<PackageReference Update="SkiaSharp.Views" Version="2.80.0" />
 	<PackageReference Update="SkiaSharp" Version="2.80.0" />
-	<PackageReference Update="SkiaSharp.NativeAssets.Linux" Version="2.80.0" />
+    <PackageReference Update="SkiaSharp.NativeAssets.Linux" Version="2.80.0" />
+    <PackageReference Update="SkiaSharp.HarfBuzz" Version="2.80.0" />
```

Comment:
Maybe there is additional space here?

Reply:
It's Visual Studio doesn't respect spaces vs tab preference in those files I believe. Will fix.